packages/suite/src/components/suite/modals/metadata/MetadataProvider/index.tsx

```diff
@@ -1,26 +1,44 @@
 import React, { useState } from 'react';
 import styled from 'styled-components';
-import { Modal, Button } from '@trezor/components';
+import { Modal, P, Button, variables, colors } from '@trezor/components';
+
 import { Translation } from '@suite-components';
 import { useActions } from '@suite-hooks';
 import * as metadataActions from '@suite-actions/metadataActions';
 import { Deferred } from '@suite-utils/deferred';
 import { MetadataProviderType } from '@suite-types/metadata';
-import { resolveStaticPath } from '@suite-utils/nextjs';
+
+const { FONT_SIZE, FONT_WEIGHT, SCREEN_SIZE } = variables;
 
 const Buttons = styled.div`
     display: flex;
+    justify-content: center;
     flex-direction: row;
-    justify-content: space-around;
 `;
 
+// todo: can't use button from @trezor/components directly, probably inconsistent design again
+// background-color is not even in components color palette
+
 const StyledButton = styled(Button)`
     padding: 10px;
-    flex-basis: 45%;
+    margin: 0 16px;
+    font-size: ${FONT_SIZE.NORMAL};
+    color: ${colors.BLACK25};
+    background-color: #f7f7f7;
+    font-weight: ${FONT_WEIGHT.DEMI_BOLD};
+    height: 42px;
```

Comment:
huh, these 2 lines looks like that designer wants us to use primary button, just with different colors?

Reply:
font-size: ${FONT_SIZE.NORMAL} is also straight from primary button. But anyway, it seems our designer come up with new button variant which wasn't properly introduced in design components, so I am gonna say a bit of improvisation is fine in this case 😁 
 https://app.zeplin.io/project/5ee87c00f6af719a645d14c3/styleguide/components?seid=5ee87ed1739bf5b789b32a45

Reply:
primary probably needs more css lines... there is hover effect, you need to change color.

Reply:
sure, use whatever fits your case most. If this variant pops out again we can move it to components, but for now I feel this is just ad-hoc design and there is no need to bring it to components just to remove it few months later.